Extensions/TileMap/JsExtension.js

```diff
@@ -342,6 +358,49 @@ module.exports = {
       .getCodeExtraInformation()
       .setFunctionName('getLayerIndex');
 
+    object
+      .addCondition(
```

Comment:
You can now use `addExpressionAndConditionAndAction` to declare the all 3 (check BitmapText/JsExtension.js for an example).

Reply:
nice! :+1:  I will refactor it :)

Reply:
Updated now :)